hadoop-ozone/dist/src/main/smoketest/balancer/testBalancer.robot

```diff
@@ -95,15 +95,16 @@ Get Uuid
 Close All Containers
     FOR     ${INDEX}    IN RANGE    15
         ${container} =      Execute          ozone admin container list --state OPEN | jq -r 'select(.replicationConfig.replicationFactor == "THREE") | .containerID' | head -1
-        EXIT FOR LOOP IF    "${container}" == ""
-                            Execute          ozone admin container close "${container}"
+        EXIT FOR LOOP IF    "${container}" == "${EMPTY}"
+                            ${message} =    Execute And Ignore Error    ozone admin container close "${container}"
+                            Run Keyword If    '${message}' != '${EMPTY}'      Should Contain   ${message}   is in closing state
         ${output} =         Execute          ozone admin container info "${container}"
                             Should contain   ${output}   CLOS
```

Comment:
The reason of `CLOS` is to include both `CLOSING` and `CLOSED`?

Reply:
Yes for this test we don't need to wait for the containers to be completely closed (this may take too long). Therefore we are happy with both “closed” and “closing” statuses.

Reply:
Thank you for the explanation.